tests/test_api.py

```diff
@@ -17,3 +18,40 @@ def test_simple(tmpdir):
         TESTDATA_DIR/"word_count/correct/output",
         tmpdir/"output",
     )
+
+
+def test_bash_executable(tmpdir):
+    """Run a MapReduce job written in Bash."""
+    with tmpdir.as_cwd():
+        madoop.mapreduce(
+            input_dir=TESTDATA_DIR/"word_count/input",
+            output_dir="output",
+            map_exe=TESTDATA_DIR/"word_count/wc_map.sh",
+            reduce_exe=TESTDATA_DIR/"word_count/wc_reduce.sh",
+        )
+    utils.assert_dirs_eq(
+        TESTDATA_DIR/"word_count/correct/output_bash",
```

Comment:
Shouldn't the output be the same as with the Python word count?

Reply:
They were different I think because there were some trailing white spaces in the input and the python mapper and reducer didn't count those whitespaces toward the word count whereas the bash ones counted them.

Reply:
I think a better solution would be to modify the input and remove the trailing space.  Then, delete the duplicate output files.

Reply:
After removing the trailing white spaces in the input, the contents of the output files match, however, the tests are still saying they are different because I think some of the tabs in the output for the bash scripts are different sizes. I could change the tests so instead of doing `filecmp`, we extract and compare the actual contents? @awdeorio

Reply:
Just change one of the reduce programs so that the whitepace matches

Reply:
Right, don't know why I didn't think of that 😂. Will change it later today.